src/zabapgit_services.prog.abap

```diff
@@ -2,8 +2,9 @@
 *&  Include           ZABAPGIT_SERVICES
 *&---------------------------------------------------------------------*
 
+INCLUDE zabapgit_services_git.
+INCLUDE zabapgit_services_trans_branch.
```

Comment:
What if I move the class `lcl_transport_to_branch` to already existing include `zabapgit_transport`